README.md

```diff
@@ -1,76 +1,115 @@
-# Webpack Linters Template
+# 📗 Table of Contents
 
-This is a template repository for setting up a project with Webpack and linters (ESLint and Stylelint) configured. It provides a solid foundation for building JavaScript and CSS projects while ensuring code quality and adherence to coding standards.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ ](#faq)
+- [📝 License](#license)
 
-## Features
+# 📖 Leaderboard <a name="about-project"></a>
 
-- Webpack configuration for bundling JavaScript and CSS files
-- ESLint for JavaScript linting with a recommended configuration
-- Stylelint for CSS linting with a recommended configuration
-- Pre-commit hooks to run linters before committing changes
-- Deployment script for deploying the project to GitHub Pages
+The leaderboard is board for displaying the ranking of the leaders in a competitive event built with HTML, CSS and Javascript.
 
-## Getting Started
+## 🛠 Built With <a name="built-with"></a>
 
-To use this template and set up your project, follow these steps:
+### Tech Stack <a name="tech-stack"></a>
```

Comment:
- [ ] The html link which you provided under this section is currently broken as it leads to a page which throws up a `page not found` error. I kindly suggest you provide the appropriate link :+1:
- [ ] You also used the same link for the `Css & JavaScript`. I kindly suggest you provide the appropriate link for the javascript :+1: :sparkles: